tutorials/RandomNumberGeneration/RandomNumberGenerationTutorial.ipynb

```diff
@@ -191,9 +191,13 @@
     "In exercise 3, we generated numbers in the range $[0, 2^N-1]$ $(1 \\leq N \\leq 10)$. Now let's create an operation that will return a random number in the range $[min, max]$. \n",
     "\n",
     "**Input:** \n",
-    "Two integers $min$ and $max$ ($1 \\leq min \\leq max \\leq 2^{10}-1$).\n",
+    "A $range$ tuple of the form ($min$, $max$) such that ($0 \\leq min \\leq max \\leq 2^{10}-1$).\n",
     "\n",
-    "**Goal:** Generate a random number in the range $[min, max]$ with an equal probability of getting each of the numbers in this range."
+    "**Goal:** Generate a random number in the range $[min, max]$ with an equal probability of getting each of the numbers in this range.\n",
+    "\n",
+    "> Useful Q# documentation: \n",
+    "> * [`Math` namespace](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.math)\n",
```

Comment:
I like the link to `BitSizeI`, but since it's the only Math function we're going to use in this task, we probably don't need the link to `Math` namespace, the link to the function itself will be sufficient.

Reply:
Done